### PR TITLE
docs: Parameters example

### PR DESCRIPTION
**Pull request by amirkaws**
Issue number: #286 

### Changes

Introduce an example for Parameters utility.

### User experience

This changes added an example solution on how to use Parameters utility

## Checklist

Please leave checklist items unchecked if they do not apply to your change.

* [x] [Meets tenets criteria](https://awslabs.github.io/aws-lambda-powertools-dotnet/tenets)
* [x] I have performed a self-review of this change
* [x] Changes have been tested
* [x] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-dotnet/blob/develop/.github/semantic.yml)

<details>
<summary>Is this a breaking change?</summary>

**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

### CODE REVIEW

File: examples/Parameters/README.md
Comment (sliedig) [inline examples/Parameters/README.md:1]: Please add instructions on how to deploy dependencies

File: examples/Parameters/template.yaml [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+Mappings:
+  Constants:
+    DynamoDB:
+      SingleParamTableName: PowertoolsParametersSingle
+      SingleParamId: my-parameter
+      MultiParamTableName: PowertoolsParametersMultiple
+      MultiParamId: my-hash-key
+    SSM:
+      SingleParamName: /powertools/path/prefix/parameter
```
Comment (sliedig) [inline examples/Parameters/template.yaml:17]: rename to /powertools-examples/path/prefix/parameter

File: examples/Parameters/template.yaml [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+      OtherParamName: /powertools/path/prefix/other-parameter
```
Comment (sliedig) [inline examples/Parameters/template.yaml:18]: /powertools-examples/path/prefix/other-parameter

File: examples/Parameters/template.yaml [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+      MultiParamPrefix: /powertools/path/prefix
```
Comment (sliedig) [inline examples/Parameters/template.yaml:19]: /powertools-examples/path/prefix

File: examples/Parameters/template.yaml [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Example project for Powertools for AWS Lambda (.NET) Parameters utility
+
+
+Mappings:
+  Constants:
+    DynamoDB:
+      SingleParamTableName: PowertoolsParametersSingle
```
Comment (sliedig) [inline examples/Parameters/template.yaml:12]: rename to PowertoolsParametersExampleSingle

File: examples/Parameters/template.yaml [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+      SingleParamId: my-parameter
+      MultiParamTableName: PowertoolsParametersMultiple
```
Comment (sliedig) [inline examples/Parameters/template.yaml:14]: rename to PowertoolsParametersExampleMultiple

**Review by sliedig — CHANGES_REQUESTED** [review]
@amirkaws just a couple of comments, nothing major. Just need to provide a bit more context for the demo

File: examples/Parameters/template.yaml (hunk 3 above)
Comment (amirkaws) [reply examples/Parameters/template.yaml]: renamed it to /powertools-parameters-example/path/prefix

File: examples/Parameters/template.yaml (hunk 5 above)
Comment (amirkaws) [reply examples/Parameters/template.yaml]: done

**Review by hjgraca — APPROVED** [review]

File: examples/Parameters/README.md
Comment (amirkaws) [reply examples/Parameters/README.md:1]: done

**Review by hjgraca — APPROVED** [review]

**Comment by amirkaws** [comment]
> @amirkaws just a couple of comments, nothing major. Just need to provide a bit more context for the demo

@sliedig fyi, I have added a section in readme for the additional resources are created and where to find them, also addressed other comments.

**Review by sliedig — APPROVED** [review]

**Pull request merged**